### ISSUE TITLE
BUG: type popups on .tsx files imported from github repo

### ISSUE THREAD

**Issue by ntucker** [stackblitz/webcontainer-core#56]
---
name: Bug report
about: Create a report to help us improve
title: ''
labels: ''
assignees: ''

---

**Describe the bug**
Typescript types highlighting works on the *.ts files, but the *.tsx files it does not. I thought it might be due to use of linaria, but even non-linaria .tsx files there is no highlighting.

**Link to the blitz that caused the error**
https://stackblitz.com/edit/github-8auner?file=src%2FBoundary.tsx

**To Reproduce**
1. Go to https://stackblitz.com/edit/github-8auner?file=src%2FBoundary.tsx
2. Hover over 'children' variable

**Expected behavior**
Hover context popup with its type

**Version of webcontainer**
Hash: 7286dbe8d47d3473cde7be488b7d048fb59b05f2

**Screenshots**

**Desktop (please complete the following information):**
Browser name  = Chrome 
Full version  = 90.0.4430.212
Major version = 90
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.212 Safari/537.36
performance.memory = {
  totalJSHeapSize: 159765627,
  usedJSHeapSize: 156720619,
  jsHeapSizeLimit: 4294705152
}

**Additional context**

**Comment by KwintenP** [comment]
Hi!

🙏 for reporting. This does seem like a bug on our end!

**Comment by SamVerschueren** [comment]
@ntucker The tsx type issue is fixed and it now gives proper intellisense and auto complete.

For your example in particular I did notice an other issue though. We currently run with the TypeScript 4.0 services, and for this it apparently needs TypeScript 4.1 at least. We have an internal ticket to track the upgrade of TypeScript on our end.

![image](https://user-images.githubusercontent.com/1913805/127308837-d9133e30-09e7-40d9-8847-57dd154edb74.png)

Closing this as the original issue was resolved.
